app/src/main/java/in/testpress/testpress/authenticator/ResetPasswordVerificationActivity.java

```diff
@@ -0,0 +1,70 @@
+package in.testpress.testpress.authenticator;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.afollestad.materialdialogs.GravityEnum;
+import com.afollestad.materialdialogs.MaterialDialog;
+
+import javax.inject.Inject;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import in.testpress.testpress.Injector;
+import in.testpress.testpress.R;
+import in.testpress.testpress.core.TestpressService;
+import in.testpress.testpress.models.ResetPassword;
+import in.testpress.testpress.util.SafeAsyncTask;
+
+public class ResetPasswordVerificationActivity extends AppCompatActivity {
+
+    @Inject TestpressService testpressService;
+    @InjectView(R.id.et_useremail) EditText useremailText;
+    @InjectView(R.id.b_reset_password) Button resetButton;
+    public int resetErrorMessage;
+    public String ok;;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_reset_password_verification);
+        Injector.inject(this);
+        ButterKnife.inject(this);
+        resetErrorMessage = R.string.reset_error_message;
+    }
+
+    @OnClick(R.id.b_reset_password) public void reset(){
+
```

Comment:
No validation being done? Even if i enter a non email string, it will do a server request.